app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -345,19 +345,19 @@ msgstr "Crynodeb"
 msgid "Check your answers and submit"
 msgstr "Gwiriwch eich atebion a'u cyflwyno"
 
-#: templates/summary.html:18
-msgid "Check your answers before submitting"
-msgstr "Gwiriwch eich atebion cyn eu cyflwyno"
+#: templates/summary.html:17 templates/summary.html:28
+msgid "You can check your answers below"
+msgstr "Gallwch wirio eich atebion isod"
 
 #: templates/summary.html:27
-msgid "Check your answers before continuing to the next section"
-msgstr "Gwiriwch eich atebion cyn parhau i'r adran nesaf"
+msgid "This section is now complete"
+msgstr "Mae'r adran hon nawr wedi'i chwblhau"
```

Comment:
These are the old strings. The template file must have been generated on v3 without the census-rehearsal changes.

Reply:
I've now loaded the the updated messages.pot to crowdin

Reply:
Have downloaded from crowdin and updated the static content.